dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -2060,6 +2093,10 @@ private boolean checkTargetPoolSize(int poolSize) {
     private void reportSystemProperties() {
         StringBuilder sb = new StringBuilder();
 
+        sb.append("\n coreThreads: ").append(String.format("%6d", Integer.valueOf(coreThreads)));
+        sb.append(" maxThreads: ").append(String.format("%6d", Integer.valueOf(maxThreads)));
+        sb.append(" startupPoolSize: ").append(String.format("%6d", Integer.valueOf(startupPoolSize)));
+
```

Comment:
Did add `startupHangCpuUtilThreshold` for some reason?

Reply:
Good point - just pushed an 'amend' to the commit to add that line to the trace.